pandas/core/window/rolling.py

```diff
@@ -2288,7 +2312,25 @@ def var(
         "scipy.stats.skew : Third moment of a probability density.\n",
         template_see_also,
         create_section_header("Notes"),
-        "A minimum of three periods is required for the rolling calculation.\n",
+        dedent(
+            """
+        A minimum of three periods is required for the rolling calculation.\n
+        """
+        ).replace("\n", "", 1),
```

Comment:
can we remove this replace too?

Reply:
ah I see, thanks - maybe we can remove the rest as part of a separate PR?

Reply:
Sure